pkg/utils/podIP.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+func PodIP(podIpBefore [3]string, podNs string, podLabels string, client *kubernetes.Clientset) bool {
```

Comment:
checkPodIP (naem of func, name of file) and no. of args

Reply:
or comparePodIP